groclient/client.py

```diff
@@ -698,6 +698,58 @@ def get_geojson(self, region_id, zoom_level=7):
         """
         return lib.get_geojson(self.access_token, self.api_host, region_id, zoom_level)
 
+    def get_descendant(
+        self,
+        entity_type,
+        entity_id,
+        distance=None,
+        include_details=True,
+    ):
+        """Look up details of all entities level contained by an item/metric/regions.
+
+        Given any region by id, get all the descendant regions that are of the specified level.
```

Comment:
It seems like this line in the docstring doesn't apply to this method?

Reply:
Fix it